src/idl_gen_rust.cpp

```diff
@@ -1115,7 +1115,7 @@ class RustGenerator : public BaseGenerator {
   // Generates a fully-qualified name getter for use with --gen-name-strings
   void GenFullyQualifiedNameGetter(const StructDef &struct_def,
                                    const std::string &name) {
-    code_ += "    pub const fn get_fully_qualified_name() -> &'static str {";
```

Comment:
I think we need to generate `get_fully_qualified_name()` with the deprecation warning too

Reply:
I thought this api was added recently and is not public yet.

Reply:
We try and encourage people to work from HEAD so its probably a good idea